### PR TITLE
Adding an icon to the app

### LINKED ISSUE #30: Add appicon

### PR DESCRIPTION
**Pull request by chacaa**
Adds an icon to the app

### Related Github project cards:
* [Add appicon](https://github.com/chacaa/DayDreaming/issues/30) (resolves #30)

### UI preview:
* #### Icon design
daydreaming_icon_720

* #### UI
UI

/cc @xmartlabs/android

### CODE REVIEW

File: app/src/main/java/com/xmartlabs/daydreaming/ui/DiagonalLayoutView.java [hunk 1]
```diff
@@ -14,6 +14,7 @@
 import android.widget.LinearLayout;
 import android.widget.TextView;
 
+import com.github.florent37.diagonallayout.DiagonalLayout;
```
Comment (bryant1410) [inline app/src/main/java/com/xmartlabs/daydreaming/ui/DiagonalLayoutView.java:17]: Unused import

File: app/build.gradle [hunk 2]
```diff
@@ -243,6 +243,7 @@ dependencies {
     compile "io.reactivex.rxjava2:rxjava:${RX_JAVA_VERSION}"
     //compile 'net.ltgt.dagger:bullet:1.0-SNAPSHOT' // Loaded from JitPack
     compile "org.parceler:parceler-api:${PARCELER_VERSION}"
+    compile files('libs/YouTubeAndroidPlayerApi.jar')
```
Comment (bryant1410) [inline app/build.gradle]: Repeated changes in this PR.

**Review by bryant1410 — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 2 above)
Comment (mirland) [reply app/build.gradle]: Also delete the jar file

**Review by mirland — APPROVED** [review]

**Review by mirland — CHANGES_REQUESTED** [review]
Please, change the pull request name

**Review by diegomedina248 — APPROVED** [review]

File: app/src/main/java/com/xmartlabs/daydreaming/ui/DiagonalLayoutView.java (hunk 1 above)
Comment (chacaa) [reply app/src/main/java/com/xmartlabs/daydreaming/ui/DiagonalLayoutView.java:17]: I use it when I do the bind of `diagonalSeparator`.

**Review by bryant1410 — APPROVED** [review]

**Review by mirland — APPROVED** [review]

**Review by michaelEllisUy — APPROVED** [review]

**Pull request merged**